content/2019/05/24/generate.rst

```diff
@@ -0,0 +1,187 @@
+author: sondra
+tags: [Conferences, Code, CSS, Workshop]
+image:
+  - src: 'blog/2019/houdini-bunny.jpg'
+headline:
+  - tagline: |
+      What I learned at GenerateConf
+summary: |
+  As my first foray
+  into the world of web design conferences,
+  GenerateConf 2019
+  in New York
+  did not disappoint.
+
+
+The Scary & the Inspiring
+=========================
+
+Josh Clark
+----------
+
+AI is your new design material
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After listening to Josh Clark’s description
+of how to use AI to amplify human experience —
+give machines the time-consuming,
+repetitive tasks they do best —
+I’m maybe not as off-put by AI as I was before.
+There are scary bias problems with AI
+(like this from a recent article in `Forbes`_).
+But if we can use those results
+to help expose and address bias as Josh suggested,
+I’m all in.
+
+.. _Forbes: https://www.forbes.com/sites/bernardmarr/2019/01/29/3-steps-to-tackle-the-problem-of-bias-in-artificial-intelligence/
+
+Miriam Suzanne
+--------------
+
+Dynamic CSS: Layouts & Beyond
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+I attended Miriam Suzanne’s workshop
+on Advanced CSS and her talk on Dynamic CSS.
+It was a blizzard of useful information
+about CSS Grid, variables, interactions between CSS and JS,
+and much more.
+If you're curious,
+check out the full video.
+If you were at either Generate or Smashing Conf
+and still want more,
+Miriam answers attendee questions in `Smashing CSS Followup`_.
+
+.. _Smashing CSS Followup: /2019/05/17/smashing/
+
+Sam Richard
+-----------
+
+Design System Magic with Houdini
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sam Richard handed me a cute little rabbit sticker
+and introduced Houdini.
+Both blew my mind.
+
+Sophia Prater
+-------------
+
+UX for Lizard Brains
+~~~~~~~~~~~~~~~~~~~~
+
+I was sad to miss Sophia Prater’s talk and workshop,
+but I had the opportunity to meet her
+and really appreciated her offer
+to discuss my recent experiments
+in Object Oriented UX.
+It was challenging
+to think about design like a data model,
+but it was also useful —
+and cut down noticeably
+on project planning time.
+
+Adekunle Oduye
+--------------
+
+Rapid Prototyping with Gatsby
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adekunle Oduye gave us an overview
+of rapid prototyping with Gatsby —
+create a design system,
+set up Gatsby,
+and start prototyping with code.
+It’s a super efficient way
+to communicate designs to developers.
+
+Aaron Irizarry
+--------------
+
+Groundwork: Building Foundations for High-Performing Teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aaron Irizarry confirmed my anecdotal experience
+about the factors that make a high performance team:
+create a psychologically safe work environment,
+avoid micromanagement and low standards,
+allow autonomy,
+provide clear expectations, stakes, goals, and evaluations,
+and teach people how to think.
+Thanks, OddBird,
+for challenging me to do my best work
+in a thoroughly supportive space. 🎉
```

Comment:
Feel free to remove the `🎉` -- I just wanted a bit more "fun" or informal personality here. Maybe 👏?